### DISCUSSION TITLE
Invite codes history missing

### DISCUSSION THREAD

**Discussion by IamC8** [MarshalX/atproto#163] [Questions] [answered]
Hi. After recent update no invite history for most accounts.
Returns codes=[]
```
res = client.com.atproto.server.get_account_invite_codes(
models.ComAtprotoServerGetAccountInviteCodes.Params(
createAvailable=True,
includeUsed=True)
)
```

**Comment by MarshalX** [comment]
Hi. But do you have it in UI? On bsky.app

**Comment by MarshalX** [comment]
That works perfectly for me

**Comment by IamC8** [comment]
Nope. I have 5 accounts, all approx 70 days old. Only one shows invite history, and I have sent invites to others from all of them. On the app, desktop and sdk , its all the same. Very strange.

**Comment by MarshalX** [comment]
Yeah strange. But not a SDK problem as I can understand

**Comment by IamC8** [comment]
I agree, sdk is the same as web, looked at the calls in the browser. All the same.
Very strange.
I have tried with different User-Agents, but the results are the same. No history on most.

**Comment by MarshalX** [comment, marked as answer]
Probably that's why https://bsky.app/profile/intrnl.bsky.social/post/3k7uql33z3y27

https://github.com/bluesky-social/atproto/pull/1642

![image](https://github.com/MarshalX/atproto/assets/15520314/bfe2c4d0-2a0d-451d-a086-548a1031ca04)
